.gitignore

```diff
@@ -11,5 +11,9 @@ thumbs.db
 *.rs.bk
 *.swp
 
+# IDE / Editor files
+*.iml
+/.idea
```

Comment:
Not really, just thought this folder would only appear in the top level directory. But you're right, I'll fix that.